# DEF-D Fysieke Ontwerpopdracht 2 invul template

## Groep informatie

Aan het einde van deze ontwerpopdracht lever je als groep één gezamenlijk notebook in (als in *PR* op GitHub en de link op Brightspace). Zet jullie groepsnummer, studienummers en namen in onderstaande tabel zodat wij zeker weten dat het groepsvormen goed is gegaan.

| Groep: 28     |  |
| :-------------|:-------------|
| Christoph Smit| 6494056 |
| Lieke Boontjes| 6543715 |
| Frederik Roerink| 6550223 |
| Planning Groep: 28     |10:55  |
|---|---|
| Condensator klaar | 12:30 - 12.45 |
| Sensor klaar | 17:00 - 17:30|
| groep aanwezig 1| 10.45 |
| groep aanwezig 2| 13.30 |
| Pauze 1| 15:00 - 15:10 |
| Lunch| 12:30 - 13:30 |
| Pauze 2| 16:00 - 16:10 |

## *Opdracht 1*: Sanity check 3

#### Voeg hier een foto van de schets van je plaatcondensator in. Geef daarbij ook duidelijk de waarde van $A$ en $d$ aan.
<img src="schets_condensator.jpeg" alt="schets student 1" width="400"/>

In [9]:
# Schrijf hier een python functie die de capaciteit van een plaatcondensator uitrekent. Zorg dat de functie
# als input de ontwerpparameters van de vlakke plaat formule accepteert (oppervlakte, afstand en diëlektrische
# constante) en als output de capaciteit van de vlakke plaat condensator met die ontwerpparameters.
# Je gaat deze formule de rest van de dag gebruiken: controleer hem goed!
epsilon_0 = 8.854187817e-12 # F/m

def capaciteit_plaatcondensator(A, d, epsr = 1.0):
    return epsilon_0 * epsr * A / d
# Optioneel: schrijf je functie zodanig dat je arrays als input kan geven en (dus) ook arrays als output.

In [ ]:
# Gebruik je formule om uit te rekenen wat je verwacht dat de capaciteit van jullie vlakke plaat condensatoren
# gaat zijn.

C_F = [capaciteit_plaatcondensator(2.0, 0.001), capaciteit_plaatcondensator(4.7, 0.001),capaciteit_plaatcondensator(1.2, 0.001)]
C = [c * 1e12 for c in C_F] # omzetten van F naar pF

print(f"Berekende capaciteit plaatcondensator groep 28 is {C[0]:.1f} pF")
print(f"Berekende capaciteit plaatcondensator groep 28 is {C[1]:.1f} pF")
print(f"Berekende capaciteit plaatcondensator groep 28 is {C[2]:.1f} pF")

C_real = capaciteit_plaatcondensator(0.01, 0.001)
C_real_pf = C_real * 1e12 # omzetten van F naar pF
print(f"Berekende capaciteit plaatcondensator groep 28 is {C_real_pf:.1f} pF")


Berekende capaciteit plaatcondensator groep 28 is 17708.4 pF
Berekende capaciteit plaatcondensator groep 28 is 41614.7 pF
Berekende capaciteit plaatcondensator groep 28 is 10625.0 pF
Berekende capaciteit plaatcondensator groep 28 is 88.5 pF


#### Voeg hier een foto van je plaatcondensator en van de meting op de scope in. Dus 2 foto's.
![scope meting 1](opstelling_condensator_1.jpeg)
![scope meting 2](opstelling_condensator_2.jpeg)

### Meting capaciteit
Vul onderstaande tabel in. Bij Sanity check 2 heb je afgeleid hoe je van berekende capaciteit naar berekend voltage gaat en terug. 


| Namen         | Berekende capaciteit   | Berekend Voltage | Gemeten Voltage | Gemeten capaciteit   |
|-------------|-------------:|-------------:|-------------:|-------------:|
| groep 28| 88.5 pF |0.45 V | 0.75 V |149 pF |

## *Opdracht 2:* Synthese.

#### Voeg hier voor een foto van je schets(en) van minimaal drie sensoren in.
![drie schetsen](tudelftLogo.png)


## Opdracht 3: Welke sensor gaan jullie maken?

Pas deze tekst aan: Wij gaan een sensor maken die X meet.

## Opdracht 4: Uitgewerkte schets (afmetingen!) en plot van de ontwerpgrafiek.

#### Voeg hier een foto van jullie uitgewerkte  schets van de sensor die jullie gaan maken in
![uitgewerkte schets](tudelftLogo.png)

In [ ]:
# Plot hier de verwachte ontwerpgrafiek van jullie sensoren.

## Opdracht 5: Invullen tabel LTSpice.

|         | Capaciteit waarde (pF)  | LTSpice berekend output Vpp (mV)|
|-------------|-------------:|-------------:|
| minmale capaciteit| ? |? |
| gemiddelde capaciteit| ? |? |
| maximale capaciteit| ? |? |

## Opdracht 6: Foto's werkende opstelling.

#### Voeg hier de foto's van je sensoren in actie in.
![meting 1](tudelftLogo.png)
![meting 2](tudelftLogo.png)

## Opdracht 7: Plot de kalibratiegrafiek.

In [ ]:
# Plot hier je gemeten kalibratiegrafiek en plot in dezelfde figuur ook je verwachte ontwerpgrafiek.

## Opdracht 8: Vul de tekst aan.

Onze sensor kan nog verbetered worden door de volgende verbeteringen (minimaal 2):

* verbetering 1
* verbetering 2

## Opdracht 9: Uiteindelijke ontwerp en kalibratiegrafiek.

#### Voeg hier een foto van je uiteindelijke sensor in.
![foto sensor](tudelftLogo.png)

In [ ]:
# Plot hier je oorspronkelijke en je uiteindelijke kalibratiegrafieken, fit daar een bijpassende lijn door
# en zet ook de verwachte ontwerpgrafiek erbij

## Opdracht 10: Vul hieronder de conclusie in.

Vul hier de conclusie in

## Opdracht 11: Kopieer hieronder de leerdoelen.
1. Het kunnen gebruiken van de eigenschappen van een elektrische component in een ontwerpproces om een fysische grootheid te meten.
2. Kennis hebben van gedrag van elektrische componenten en meetopstellingen.
3. Onderdelen van een datasheet zoals een kalibratiegrafiek opstellen, gevoeligheid en onzekerheid beschrijven.

kopieer hier de cel(len) waaruit blijkt dat jullie leerdoel 1 beheersen.

kopieer hier de cel(len) waaruit blijkt dat jullie leerdoel 2 beheersen.

kopieer hier de cel(len) waaruit blijkt dat jullie leerdoel 3 beheersen.